### ISSUE TITLE
allow ESC to delete an empty card with no children from the editing mode

### ISSUE THREAD

**Issue by jayarmstrong** [gingko/client#143]
*Feature request*

When editing, I occasionally end up focused on an (empty) card I didn't mean to create. Hitting escape would be a quick, intuitive action to abandon the empty card. Since there's no way to delete cards using the keyboard, this would also save me from having to use the mouse at all.

**Comment by AdrianoFerrari** [comment]
Good point, esc on new empty should delete it. Thanks for contributing.

FYI, you can delete cards with shortcut: Ctrl+backspace
